package.json

```diff
@@ -87,6 +87,7 @@
     "null-loader": "^3.0.0",
     "object-hash": "^2.0.3",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
+    "p-timeout": "^5.0.0",
```

Comment:
did you actually end up using this package? Because I can't see it being imported anywhere

Reply:
no, we didnt. removing